scripts/run-batch-deletes.sh

```diff
@@ -27,10 +27,17 @@ while true; do
     printf "Deleting..."
     # DBURI and MAX_AGE should be in the environment
     DELETED=$(python ./manage-db.py -d "$DBURI" cleanup "$MAX_AGE" 2>/dev/null | awk '/Total/ {print $3}')
+    ORIG_DELETED=$DELETED
+    # Sometimes DELETED ends up not being set for some reason, which breaks
+    # the `expr` expressions below, and gets the script stuck in a loop.
```

Comment:
Maybe because manage-db.py failed, or failed to produce the right output?
Should we check $? here?

Reply:
That's possible...the logs didn't make it look that way, but it's hard to be certain. Let's do this!